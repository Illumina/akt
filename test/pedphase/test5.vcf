chr20	1197135	rs13044333	T	C	711	PASS	.	GT	0|1	0|0	1|1
chr20	1197291	rs6104732	T	C	276	PASS	.	GT:PS	1|0:1197291	1|0:1197291	0|0:.
chr20	1197325	rs6104733	T	C	229	PASS	.	GT:PS	1|0:1197291	1|0:1197291	0|0:.
chr20	1197765	rs9789836	A	G	324	PASS	.	GT	1|0	1|0	0|0
