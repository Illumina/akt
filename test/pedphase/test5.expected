#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	CephChild-12882	CephMother-12878	CephFather-12877
chr20	1197135	rs13044333	T	C	711	PASS	.	GT:ME	0|1:0	0|0:.	1|1:.
chr20	1197291	rs6104732	T	C	276	PASS	.	GT:RPS:ME	1|0:1197291:0	1|0:1197291:.	0|0:.:.
chr20	1197325	rs6104733	T	C	229	PASS	.	GT:RPS:ME	1|0:1197291:0	1|0:1197291:.	0|0:.:.
chr20	1197765	rs9789836	A	G	324	PASS	.	GT:ME	1|0:0	1|0:.	0|0:.
